sdk/communication/azure-communication-messages/src/main/java/com/azure/communication/messages/models/channels/WhatsAppMessageTemplateItem.java

```diff
@@ -118,10 +117,27 @@ public static WhatsAppMessageTemplateItem fromJson(JsonReader jsonReader) throws
             }
             WhatsAppMessageTemplateItem deserializedWhatsAppMessageTemplateItem
                 = new WhatsAppMessageTemplateItem(language, status);
-            MessageTemplateItemAccessHelper.setName(deserializedWhatsAppMessageTemplateItem, name);
+            deserializedWhatsAppMessageTemplateItem.name = name;
             deserializedWhatsAppMessageTemplateItem.kind = kind;
             deserializedWhatsAppMessageTemplateItem.content = content;
             return deserializedWhatsAppMessageTemplateItem;
         });
     }
+
+    /*
+     * The template's name.
+     */
+    @Generated
+    private String name;
+
+    /**
+     * Get the name property: The template's name.
+     *
+     * @return the name value.
+     */
+    @Generated
+    @Override
+    public String getName() {
+        return this.name;
+    }
```

Comment:
Newly generated code is not wrong. Seems service previously customized using accessor pattern.